pkgs/development/python-modules/nbclient/default.nix

```diff
@@ -5,12 +5,16 @@
 
 buildPythonPackage rec {
   pname = "nbclient";
-  version = "0.5.13";
+  version = "0.6.2";
+  # As of version 0.6.2, there is both a setup.py and pyproject.toml. However,
+  # the pyproject.toml does not appear to be the main entry point.
```

Comment:
```suggestion
```
Useless comment, we don't need that.

Reply:
How is this a useless comment? This exact question came up multiple times here https://github.com/NixOS/nixpkgs/pull/171563#event-6552283306. There is legitimate reason to explain this with a comment.

Reply:
I think the question only came up because setuptools is the default format which we usually don't even set. In the future there is also little to no intensive to change format unless upstream fully switches to another one. 

Also I don't think we should add a comment for every question during review or every decision during packaging. We need to add comments to places where it is really important and we do something counter intuitive and unusual to know in the future that it isn't a code smell.